NorthstarDedicatedTest/dllmain.cpp

```diff
@@ -67,6 +77,96 @@ void WaitForDebugger(HMODULE baseAddress)
 			Sleep(100);
 	}
 }
+bool LoadPlugins() {
+
+	std::vector<fs::path> paths;
+
+	std::string pluginPath = GetNorthstarPrefix() + "/plugins";
+
+	// ensure dirs exist
+	fs::create_directories("plugins");
```

Comment:
IIRC, I think this will throw an exception if it fails, e.g. when TF2 is installed with EA Desktop into a read-only folder. We should just skip plugin loading if the dir doesn't exist; it's not Northstar's place to create it.

Reply:
Aslo forgot to update the folder path here. Oops